src/ga4gh/vrs/extras/translator.py

```diff
@@ -64,39 +56,49 @@ def __init__(
         self.hgvs_tools = None
         self.from_translators = {}
         self.to_translators = {}
+    
+    # def _is_intronic(self, sv: hgvs.sequencevariant.SequenceVariant):
+    #     """Check if HGVS sequence variant is intronic"""
+    #     if isinstance(sv.posedit.pos, hgvs.location.BaseOffsetInterval):
+    #         if sv.posedit.pos.start.is_intronic or sv.posedit.pos.end.is_intronic:
+    #             return True
+    #     return False
+
+    # def _get_hgvs_refget_ac(self, sv: hgvs.sequencevariant.SequenceVariant):
+    #     """Get refget accession for hgvs sequence variant"""
+    #     # prefix accession with namespace
+    #     sequence = coerce_namespace(sv.ac)
+    #     return self._get_refget_accession(sequence)
 
-    @lazy_property
-    def _hgvs_parser(self):
-        """instantiates and returns an hgvs parser instance"""
-        _logger.info("Creating parser")
-        return hgvs.parser.Parser()
+    @staticmethod
+    def _ir_stype(a):
+        """
+        The purpose of this function is to provide a convenient way to extract the sequence type from an accession by matching its prefix to a known set of prefixes.
 
-    def _get_parsed_hgvs(self, hgvs_expr: str):
-        """Get parsed HGVS expression"""
-        if not self.hgvs_re.match(hgvs_expr):
-            return None
+        Args:
+        a (str): The accession string.
 
-        return self._hgvs_parser.parse_hgvs_variant(hgvs_expr)
+        Returns:
+        str or None: The sequence type associated with the accession string, or None if no matching prefix is found.
+        """
 
-    def _get_hgvs_refget_ac(self, sv: hgvs.sequencevariant.SequenceVariant):
-        """Get refget accession for hgvs sequence variant"""
-        # prefix accession with namespace
-        sequence = coerce_namespace(sv.ac)
-        return self._get_refget_accession(sequence)
+        prefix_dict = {
+            "refseq:NM_": "n",
+            "refseq:NC_012920": "m",
+            "refseq:NG_": "g",
+            "refseq:NC_00": "g",
+            "refseq:NW_": "g",
+            "refseq:NR_": "n",
+            "refseq:NP_": "p",
+            "refseq:XM_": "n",
+            "refseq:XR_": "n",
+            "refseq:XP_": "p",
+            "GRCh": "g",
+        }
```

Comment:
double checking: this mapping isn't available somewhere else? I did a quick search through HGVS and didn't see it, and I wouldn't expect it to be in bioutils, but just making sure

Reply:
I know. it seems like someone stuck this here to make sure we only worked with "refseq" accessions. I do think that hgvs should retain this mapping structure since it is really an hgvs thing.

Reply:
this was moved to the dataproxy classes for now. further refactoring will need to happen in another round to allow us to better organize/segregate hgvs code from vrs-python code.